Sources/tuist/tuist.docc/Articles/Tuist/Tuist - Installation.md

```diff
@@ -32,7 +32,7 @@ rtx use -g tuist@system       # Use the system's tuist as the global default
 
 > Tip: We recommend using `rtx use` in your Tuist projects to pin the version of Tuist across environments. The command will create a `.tool-versions` file containing the version of Tuist.
 
-> Important: **Tuist Cloud** (<doc:Tuist-Cloud---What>), a closed-source extension of Tuist with optimizations such as binary caching and selective testing, is distributed as a different rtx plugin, tuist-cloud. Note that by using it, you agree to the [Tuist Cloud Terms of Service](https://tuist.io/terms/).
+> Important: **Tuist Cloud** (<doc:Tuist-Cloud---What>), a closed-source extension of Tuist with optimizations such as binary caching and selective testing, is distributed as a different asdf plugin, tuist-cloud. Note that by using it, you agree to the [Tuist Cloud Terms of Service](https://tuist.io/terms/).
```

Comment:
Legally-speaking, you'd be deciding between a program, whose code is entirely open-source and with no liabilities for the community behind it, and a program that is closed source, has a company behind it, and the relationship is bound to some legal terms. A clean-cut is necessary so that users can decide at installation time. Because the cut can't be done at the plugin level since one plugin equals one tool, we need two plugins. At the system level, the CLI has the same name, `tuist`.